atlasdb-commons/src/main/java/com/palantir/util/AssertUtils.java

```diff
@@ -19,12 +19,14 @@
 import java.util.List;
 import java.util.Set;
 
+import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import com.google.common.collect.Sets;
 
 public class AssertUtils {
 
```

Comment:
I can add a new line post the log definition, this new line is complimentary to the one at the end of the class. Will leave them as is.